src/tests/class_tests/smartpeak/source/RawDataProcessor_test.cpp

```diff
@@ -2345,4 +2345,309 @@ BOOST_AUTO_TEST_CASE(emg_processor)
   std::remove(SMARTPEAK_GET_TEST_DATA_PATH("RawDataProcessor_mzML_1.featureXML"));
 }
 
+BOOST_AUTO_TEST_CASE(calculateMDVs)
+{
+  // Pre-requisites: load the parameters and associated raw data
+  map<string, vector<map<string, string>>> params_1;
+  map<string, vector<map<string, string>>> params_2;
+  load_data(params_1, params_2);
+  RawDataHandler rawDataHandler;
+
+  Filenames filenames;
+  filenames.featureXML_i = SMARTPEAK_GET_TEST_DATA_PATH("GCMS_SIM.featureXML");
+//
```

Comment:
Are these commented lines needed here?

Reply:
No longer needed, it was an unfinished attempt to to load the features from the `featureXML` file.